src/index.js

```diff
@@ -35,6 +35,15 @@ client.getCommand = function(name) {
 
 require("./commands")(client);
 
+function suggestCommands(cmd, cmdList) {
+  var suggestion = didYouMean(cmd, cmdList);
+  if (suggestion) {
+    logger.error();
+    logger.error("Did you mean", clc.bold(suggestion) + "?");
+    process.exit(1);
```

Comment:
Don't `process.exit` here - leave that to the function that calls it. To make that a little easier, I'd recommend returning `suggestion`. Then, you can make a determination of what other behavior to execute based on the return value.